src/main/tables/107_default_instant_launches.sql

```diff
@@ -0,0 +1,45 @@
+SET search_path = public, pg_catalog;
+
+CREATE TABLE IF NOT EXISTS default_instant_launches (
+    id UUID UNIQUE NOT NULL DEFAULT uuid_generate_v1(),
+
+    -- Indicates the format version of the instant launch
+    -- JSON. Should help us put together conversions between
+    -- format versions without breaking currently running
+    -- services.
+    version SERIAL,
```

Comment:
Ah, cool, this makes sense. `version` being for json-data-format versions rather than a history of changes to the set of defaults makes a lot more sense, since I guess basically this means "run `UPDATE` on the JSON column if the format isn't changing, otherwise create a new one with a new version number"